pkgs/models/src/activities/types.ts

```diff
@@ -57,12 +57,12 @@ export type ActionUser =
 
 export type ActionKey = 'Key.created' | 'Key.deleted';
 
-export type ActionGithub = 'Github.linked' | 'Github.unlinked';
+export type ActionGitHub = 'GitHub.linked' | 'GitHub.unlinked';
```

Comment:
This is already inserted in DB, since it's not released yet I can allow some breaking changes :p 
Only if it's deployed before sunday.